### DISCUSSION TITLE
rotation acts a bit odd compared to OTD's...

### DISCUSSION THREAD

**Discussion by Nowaaru** [ppy/osu#21390] [Ideas] [answered]
search queries: "tablet rotation" "rotation" "tablet driver rotation"

primarily, rotation doesn't reflect OpenTabletDriver's rotation. secondly, it also does not support negative rotation which is problematic since I do need this support as I rotate my tablet rightwards (top left corner is higher than top right corner)

if not negative support, an option to invert the axes of the tablet would also suffice. 👍 
i apologize if i shouldn't be comparing Lazer's to OTD's, i made the comparison because i remember lazer's being based off of OTD

**Comment by peppy** [comment, marked as answer]
Does 270 degree rotation not work for your case? It should match.

**Reply by Nowaaru** [reply]
270-degree+ rotation essentially flips the tablet upside down. am i doing something wrong?

**Reply by mk56-spn** [reply]
No. 270 rotation is the equivalent of -90. 180 flips it's upside down.

**Reply by Nowaaru** [reply]
You're right. It does flip it upside down. Including all of my inputs.
https://www.youtube.com/watch?v=DK0BD2l9NIU

270 degrees+ does so as well.

**Reply by plaguemask** [reply]
Your video shows the game working as intended.

Looking it up, it seems that OTD rotates the area counter-clockwise relative to the tablet for positive values (unless you have a 'left-handed' checkbox enabled). On the other hand, osu! rotates the area clockwise.

Try 90° instead.
